app/controllers/volunteer_ops_controller.rb

```diff
@@ -73,29 +76,6 @@ def restrict_by_feature_scope
     :local_only
   end
 
-  def authorize
-    # set @organisation
-    # then can make condition:
-    # unless current_user.can_edit? organisation
-    unless org_owner? || superadmin?
-      flash[:error] = 'You must be signed in as an organisation owner or ' \
-                      'site superadmin to perform this action!'
```

Comment:
Hmm, it looks like this specific message will be lost. Is it possible for this entity's pundit to raise a more specific error, and have this entity's controller rescue it with its own error message? Actually, would it be possible to define the message passed to `flash[:error]` on these pundit objects? Maybe the controller code read something like this:
```
rescue_from InvitationPolicy::NotAuthorized, with: InvitationPolicy::NotAuthorizedMessage
```